recipes/mako/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.0.4" %}
+
+package:
+  name: mako
+  version: {{ version }}
+
+source:
+  fn: Mako-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/M/Mako/Mako-{{ version }}.tar.gz
+  sha256: fed99dbe4d0ddb27a33ee4910d8708aca9ef1fe854e668387a9ab9a90cbf9059
+
+build:
+  preserve_egg_dir: True
+  entry_points:
+    - mako-render=mako.cmd:cmdline
+  script: python setup.py install
```

Comment:
Please add these arguments in the feedstock `--single-version-externally-managed --record=record.txt`.